modules/objc/generator/templates/cmakelists.template

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
Can we require lower version of CMake?
What CMake functionality is required here?

These artictles about Swift say about CMake 3.15:
- https://forums.swift.org/t/announcing-swift-support-in-cmake/24792
- https://forums.swift.org/t/bumping-cmake-minimum-to-3-15/28179

Reply:
Let me try and find a cmake 3.15 and see if it works